tools/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.csproj

```diff
@@ -35,9 +35,18 @@
       <PackageReference Include="Microsoft.Build.Framework" Version="17.8.3" />
       <PackageReference Include="Microsoft.Build.Utilities.Core" Version="17.8.3" />
       <PackageReference Include="Google.Protobuf.Tools" Version="$(NuGetProtobufVersion)" />
+      <PackageReference Include="System.Text.Json" Version="8.0.3" />
    </ItemGroup>
    <ItemGroup>
       <Compile Include="../Common/IceRpc.CaseConverter.Internal/Converter.cs" />
+      <None Include="../Common/IceRpc.Telemetry.Internal/bin/$(Configuration)/net8.0/*" Exclude="../IceRpc.Telemetry.Internal/bin/$(Configuration)/net8.0/*.exe">
+        <PackagePath>tools/</PackagePath>
+        <Pack>true</Pack>
+      </None>
+      <None Include="../Common/IceRpc.Telemetry.Internal/certs/">
```

Comment:
This is not correct, "Include" must refer to an actual file. This prevents loading the project in Visual Studio.

Reply:
Is there a better temporary solution? The certs are not going to be needed once the telemetry server is actually deployed. Purely exist for development and testing currently.